<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWithdrawDialog</class>
 <widget class="QDialog" name="DepositWithdrawDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit / Withdraw</string>
  </property>

  <!-- Balance display -->
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance: â‚¬0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- List of accounts -->
  <widget class="QListWidget" name="accountList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
  </widget>

  <!-- Amount input -->
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>

  <!-- Deposit & Withdraw buttons -->
  <widget class="QPushButton" name="depositBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>

  <widget class="QPushButton" name="withdrawBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>

  <!-- Transaction history -->
  <widget class="QListWidget" name="historyListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>460</width>
     <height>180</height>
    </rect>
   </property>
  </widget>

  <!-- Back button -->
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
